database/insertData/_helpers/dev_actions.js

```diff
@@ -3,18 +3,29 @@ GraphQL Fragment - DEVELOPMENT Actions
   - Use this to test arbitrary Actions -- but only use the trigger "DEV_TEST"
 \*/
 exports.devActions = `
-    # Create a user
+    # Send notification/email
     {
-      actionCode: "modifyRecord"
+      actionCode: "sendNotification"
       trigger: DEV_TEST
       sequence: 1
       parameterQueries: {
-        tableName: "user"
-        matchField: "username"
-        first_name: "Boba"
-        last_name: "Fett"
-        username: "js"
-        email: "nowhere@outerrim.space"
+        email: ["carl@sussol.net", "ceejaysmith@gmail.com"]
```

Comment:
probably better to change this to some sussol test email?

Reply:
Yeah, okay. The dev_actions are really meant to be just for playing round/testing and are over-written in subsequent PRs, but I've just put dummy addresses in there for now with a comment to change them: 62e7d7a4